src/Okta.Sdk/PasswordCredentialHash.cs

```diff
@@ -0,0 +1,25 @@
+﻿// <copyright file="PasswordCredentialHash.cs" company="Okta, Inc">
+// Copyright (c) 2020 - present Okta, Inc. All rights reserved.
+// Licensed under the Apache 2.0 license. See the LICENSE file in the project root for full license information.
+// </copyright>
+
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Okta.Sdk
+{
+    /// <inheritdoc/>
+    public partial class PasswordCredentialHash : Resource, IPasswordCredentialHash
+    {
+        [Obsolete("This method will deprecated in the next major version. Use WorkFactor instead.")]
+        /// <summary>
+        /// Gets and sets the <c>workFactor</c> property. 
+        /// </summary>
+        public int? WorkerFactor
```

Comment:
Shouldn't it be called "WorkFactor"?

Reply:
I added a partial class with this property to avoid a breaking change. As you can see, internally maps to `workFactor` and I also tagged it as obsolete.

Reply:
Forgot to mention, that the generation script creates a `WorkFactor` property in the generated class.